File: BANROB.py
mon = 81#**9
t = raw_input()
t = int(t)
for _ in xrange(t):
	m,p = raw_input().split();
	m = int(m)
	p = float(p)

	if m%2:
		fir=mon*(1+p**m)
	else:
		fir=mon*(1-p**m)
	fir/=(1+p)

	print fir, mon-fir

File: BANROB.cpp
#include<bits/stdc++.h>
using namespace std;

main()
{
    #ifndef ONLINE_JUDGE
        freopen("input.txt", "r", stdin);
        freopen("output.txt","w",stdout);
    #endif
    ios::sync_with_stdio(0);
    long long  t;
    cin >> t;
    cout << fixed << setprecision(6);
    while(t--)
    {
        long long  m;
        long double mon = 1000000000;
        long double p;
        cin >> m >> p;


        long double fir;

        if(m%2==1){
                    fir=mon*(1+pow(p,m));}
        else 
            fir=mon*(1-pow(p,m));
        

        fir/=(1+p);
        cout << fir << ' ' << mon-fir << '\n';

    }
	return 0;
}